docs/api/en/materials/MeshPhysicalMaterial.html

```diff
@@ -79,6 +79,16 @@ <h3>[name]( [param:Object parameters] )</h3>
 		<h2>Properties</h2>
 		<p>See the base [page:Material] and [page:MeshStandardMaterial] classes for common properties.</p>
 
+		<h3>[property:Color attenuationColor]</h3>
+		<p>
+		The color that white light turns into due to absorption when reaching the attenuation distance. Default is *white* (0xffffff).
```

Comment:
It would be nice to document what the relationship is between attenuationColor and diffuseColor /  transmission here. Why are they different?

Reply:
Because diffuse and attenuation color are part of different models and represent different things.

The diffuse color (the `color` material property in `three.js`) represents more or less an object's color. Or in other words it is the color you see when the object's surface is illuminated by pure white light.

`attenuationColor` is part of the Beer–Lambert law. Monochromatic white light travels through the material and is changed during this process. `attenuationColor` defines the color that the white light is turned into after a certain distance.